cpp/src/vertices_builder.cc

```diff
@@ -65,37 +65,37 @@ Status VerticesBuilder::validate(const Vertex& v, IdType index,
       switch (type->id()) {
       case Type::BOOL:
         if (property.second.type() !=
-            typeid(typename ConvertToArrowType<Type::BOOL>::CType)) {
+            typeid(typename TypeToArrowType<Type::BOOL>::CType)) {
```

Comment:
I want to understand that if we should add Type::LIST into the implementations of VerticesBuilder/EdgesBuilder.

Reply:
Good question, I think this can add to Builder but it's a little complicated at present. I can add a TODO tag and create a new issue.

Reply:
> Good question, I think this can add to Builder but it's a little complicated at present. I can add a TODO tag and create a new issue.

I agree, and I have no further questions.